lib/ansible/modules/packaging/os/apt.py

```diff
@@ -168,8 +168,6 @@
 
 - name: Install a list of packages
   apt:
-    name: "{{ packages }}"
-  vars:
```

Comment:
`packages` is not a valid option for `apt`, it is either `name`, `package` or `pkg`